hoki/tests/test_spec.py

```diff
@@ -1,12 +1,95 @@
 import pandas as pd
 from hoki import spec
+from hoki import load
 import numpy as np
 import pkg_resources
+import pysynphot as psp
+import pytest
 
 data_path = pkg_resources.resource_filename('hoki', 'data')
 
+#wavelengths and throughput/transmission manually read from the LICK.LICK.U file
+wl = [3100,3200,3300,3400,3500,3600,3700,3800,3900,4000,4100]
+transmission = [0.015, 0.140,0.347,0.440,0.625,0.685,0.708,0.643,0.458,0.170,0.029]
+data = load.model_output(f"{data_path}/spectra-bin-imf135_300.z002.dat")
+
 def test_dopcor():
     ngc4993 = pd.read_csv(data_path+'/ngc4993_spec_tot.dat')
     wl0_init = ngc4993.wl[0]
     spec.dopcor(ngc4993,  0.009783)
-    assert np.isclose((wl0_init-ngc4993.wl[0]), 46.4655, atol=1e-05), "The doppler correction isn't right"
\ No newline at end of file
+    assert np.isclose((wl0_init-ngc4993.wl[0]), 46.4655, atol=1e-05), "The doppler correction isn't right"
+
+
+def test_import_custom_filter():
+    # Expected values are directly from the filter file.
+    fp = spec.import_custom_filter(f"{data_path}/LICK.LICK.U.xml")
+    assert fp.name == "LICK/LICK.U", "The filter name is incorrectly imported"
+    assert (fp.wave == wl).all(), "The wavelength bins are incorrectly imported"
+    assert (np.isclose(fp.throughput, transmission)).all(),\
+           "The throughput is incorrectly imported"
+
+
+def test_load_bandpass():
+    # Test loading a build in function
+    h = spec.load_bandpass("johnson,v")
+    p = psp.ObsBandpass("johnson,v")
+    assert (h.wave == p.wave).all(),\
+           "The bins are incorrectly created using `load_bandpass`"
+    assert (h.throughput == p.throughput).all(),\
+           "The throughput is incorrect using `load_bandpass`"
+
```

Comment:
makes these separate tests under a test class.

Reply:
Separated to a specific test class